Jellyfin.Server/Program.cs

```diff
@@ -175,6 +185,11 @@ private static ServerApplicationPaths createApplicationPaths(StartupOptions opti
                 }
             }
 
+            if (string.IsNullOrEmpty(configDir))
```

Comment:
maybe do it more _right_? :)
```suggestion
            if (!string.IsNullOrEmpty(configDir))
```

Reply:
Also it should only check if it’s null, empty isn’t valid.

Reply:
Ok, third change set added.  Thank you. :)